### DISCUSSION TITLE
Check if input value has change onBlur

### DISCUSSION THREAD

**Discussion by ThomasRutzer** [react-hook-form/react-hook-form#9596] [Q&A]
is there a comfortable way to know if an inputs value has been changed between focus / blur (w/o any Component state)?

**Comment by Moshyfawn** [comment]
[`getFieldState`](https://react-hook-form.com/api/useform/getfieldstate) -> `isDirty`

**Reply by ThomasRutzer** [reply]
Thanks for your quick answer, but its not the exact solution I was looking for. What I want to know, if the users has actually changed the value of a text input when blurring it. `isDirty` flag only works for the first change, but if a user re-enters and does not change the value again, I'd like to know that.

**Reply by Moshyfawn** [reply]
You can try [`resetField`](https://react-hook-form.com/api/useform/resetfield) on `blur` event with `defaultValue: newValue`
